src/node/Launcher.ts

```diff
@@ -133,26 +133,9 @@ class ChromeLauncher implements ProductLauncher {
 
     isTempUserDataDir = false;
 
-    let chromeExecutable = executablePath;
-
-    if (channel) {
-      // executablePath is detected by channel, so it should not be specified by user.
-      assert(
-        typeof executablePath === 'string',
```

Comment:
I think we should keep the error message? to hint users we they specify both options by mistake